graylog2-server/src/main/java/org/graylog2/shared/utilities/StringUtils.java

```diff
@@ -20,9 +20,23 @@
 
 public final class StringUtils {
 
-    private StringUtils() { }
+    private StringUtils() {
+    }
 
     public static String f(String format, Object... args) {
         return String.format(Locale.ENGLISH, format, args);
     }
+
+    public static boolean containsOnce(final String mainString, final String subString) {
```

Comment:
Why do you need this method? Only usages I could find are in the test of this util class.

Reply:
Good catch! It is not needed anymore after changes made in the enterprise part.
Thank you!